includes/Plugin.php

```diff
@@ -250,6 +251,9 @@ public function register() {
 		$this->meta_boxes = new Meta_Boxes();
 		add_action( 'admin_init', [ $this->meta_boxes, 'init' ] );
 
+		$coi = new Cross_Origin_Isolation( $this->experiments );
+		add_action( 'init', [ $coi, 'init' ], 11 );
```

Comment:
`admin_init?  Don't want to run this on the frontend

Reply:
Good catch. I changed to see another action now.